### PR TITLE
Adding r-lmomco and r-spei

### PR DESCRIPTION
**Pull request by cofinoa**
Generated with conda_r_skeleton_helper from CRAN:
https://cran.r-project.org/web/packages/lmomco/index.html
https://cran.r-project.org/web/packages/SPEI/index.html

Checklist
- [X] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [X] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [X] Source is from official source.
- [X] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [X] If static libraries are linked in, the license of the static library is packaged.
- [X] Build number is 0.
- [X] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [X] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [X] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by cofinoa** [comment]
@conda-forge-admin, please ping team

@conda-forge/help-r

**Comment by cofinoa** [comment]
@conda-forge-admin, please ping team

@conda-forge/help-r

File: recipes/r-lmomco/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+    - r-goftest
+
+test:
+  commands:
+    - $R -e "library('lmomco')"           # [not win]
+    - "\"%R%\" -e \"library('lmomco')\""  # [win]
+
+about:
+  home: http://www.amazon.com/dp/1463508417/
+  license: GPL-3
```
Comment (cbrueffer) [inline recipes/r-lmomco/meta.yaml:45]:
```suggestion
  license: GPL-2.0-or-later
```

File: recipes/r-spei/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+    - r-lmomco
+
+test:
+  commands:
+    - $R -e "library('SPEI')"           # [not win]
+    - "\"%R%\" -e \"library('SPEI')\""  # [win]
+
+about:
+  home: http://sac.csic.es/spei
+  license: GPL-2
```
Comment (cbrueffer) [inline recipes/r-spei/meta.yaml:43]:
```suggestion
  license: GPL-2.0-only
```

**Review by cbrueffer — COMMENTED** [review]
Looks good; please fix the two license changes I have annotated, then I'll merge it.

**Comment by cofinoa** [comment]
@cbrueffer done! thanks!

**Pull request merged**